crates/core/src/bc_protocol/connection.rs

```diff
@@ -92,7 +92,7 @@ impl BcConnection {
     }
 
     pub fn subscribe(&self, msg_id: u32) -> Result<BcSubscription> {
-        let (tx, rx) = channel();
+        let (tx, rx) = unbounded::<Bc>();
```

Comment:
Is this not working implictly?

```suggestion
        let (tx, rx) = unbounded();
```